File: CodeForces/1901-2000/1999/A/1999A.cpp
// Problem 1999A: A+B Again?
// https://codeforces.com/contest/1999/problem/A
// Compiled with: g++ (Debian 12.2.0-14+deb12u1) 12.2.0
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 22nd, 2025
// Codeforces language used: C++23 (GCC 14-64, msys2)

#include <iostream>
#include <string>
using namespace std;

int main() {
    int t;
    cin >> t;

    for(int i = 0; i < t; i++) {
        string number;
        cin >> number;
        cout << (number[0]-'0') + (number[1]-'0') << "\n";
    }
}

File: CodeForces/1901-2000/1999/A/1999A.java
// Problem 1999A: A+B Again?
// https://codeforces.com/contest/1999/problem/A
// Compiled with: openjdk 25 2025-09-16
// Executed with: javac 25
// Linux version: Linux kernel 6.17.0-6-generic
// Submitted on: October 27th, 2025
// Codeforces language used: Java 21 64bit

import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int runs = sc.nextInt();
        for (int run = 0; run < runs; run++) {
            int n = sc.nextInt();
            System.out.println(n/10 + n%10);
        }
        sc.close();
    }
}
